extensions/GlobalMessages/GlobalMessagesG.i18n.php

```diff
@@ -1532,8 +1530,6 @@
 	<input type="radio" name="sitesearch" id="gWWW" value="" /><label for="gWWW">WWW</label>
   </div>
 </form>',
-	'group-vstf' => 'VSTF',
```

Comment:
Shouldn't be there something similar for `soap` eg. `group-soap` and `group-soap-memeber` key?

Reply:
it will be added during translations from english: https://github.com/Wikia/app/blob/PLATFORM-4806-v2/extensions/GlobalMessages/GlobalMessagesG.i18n.php#L87-L88